net/samba4/Makefile

```diff
@@ -2,19 +2,18 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=samba
-PKG_VERSION:=4.9.4
-PKG_RELEASE:=3
+PKG_VERSION:=4.9.5
+PKG_RELEASE:=1
 
 PKG_MAINTAINER:=Andy Walsh <andy.walsh44+github@gmail.com>
 PKG_LICENSE:=GPL-3.0-only
 PKG_LICENSE_FILES:=COPYING
 
 PKG_SOURCE_URL:=https://download.samba.org/pub/samba/stable/
```

Comment:
As a workaround for perhaps we can add the following mirrors to at least try to get a clean CI build?
https://ftp5.gwdg.de/pub/samba/stable/
https://ftp.yz.yamagata-u.ac.jp/pub/network/samba/
http://ftp.uni-bayreuth.de/netsoftware/samba/

Reply:
sure np